### PR TITLE
Fix version 3.1

### PR DESCRIPTION
**Pull request by dnephin**

### CODE REVIEW

File: compose/config/config.py [hunk 1]
```diff
@@ -186,11 +186,6 @@ def version(self):
         if version == '3':
             version = V3_0
 
-        if version not in (V2_0, V2_1, V3_0):
```
Comment (ebriney) [inline compose/config/config.py]: I'm surprised that there is no parsing for `3.1`version here
```
        if version == '3.1':
            version = V3_1
```
no?

File: compose/config/config.py [hunk 2]
```diff
@@ -186,11 +186,6 @@ def version(self):
         if version == '3':
             version = V3_0
```
Comment (ebriney) [inline compose/config/config.py]: I'm surprised that there is no parsing for 3.1version here

        if version == '3.1':
            version = V3_1
no?

Comment (dnephin) [inline compose/config/config.py]: No. These are aliases for converting "3" -> "3.0". There is no alias for 3.1

Comment (ebriney) [inline compose/config/config.py]: Oh yes, thanks.

**Pull request merged**